### DISCUSSION TITLE
Standalone debugger

### DISCUSSION THREAD

**Discussion by madroneropaulo** [wevm/frog-archived#134] [Ideas]
So right now I'm having [a bunch of problems ](https://github.com/wevm/frog/issues/124)with cloudflare, bindings and frog. So I'm using the frames.js debugger instead with the wrangler dev command. Unfortunately it doesn't seem that state management works with that setup. For this or any other life use cases (like debugging remote frames) I think the debugger should be  a standalone project that just receives a frame url and starts emulatig evertying.

**Comment by tmm** [comment]
Duplicate of https://github.com/wevm/frog/discussions/44
